topic_name: /camera_image
is_yuv: False
is_compressed: False
undistort: False

K: [
  [231.468, 0.0, 368], 
  [0.0, 231.468, 368], 
  [0.0, 0.0, 1.0]]

D: [-0.020286852335397013, -0.007, 0.005, -0.003]
